### PR TITLE
:tada:

### PR DESCRIPTION
**Pull request by tsunammis**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,3 +1,130 @@
-<html>
-<body>Gazr is coming ...</body>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Gazr [gather] - Opinionated way to launch common goals on projects</title>
```
Comment (achedeuzot) [inline index.html]: launch common goals ?

Comment (achedeuzot) [reply index.html]: I'd prefer "launch common tasks", it seems better to me.

File: index.html [hunk 2]
```diff
@@ -1,3 +1,130 @@
...
+        <img class="avatar" src="/images/gazr.svg" alt="Gazr">
+        <p><strong>gazr</strong>, pronounced "Gather", is an opinionated way to launch <strong>common goals</strong> on your project.</p>  
+    </header>
+    <section>
+        <div class="box">
+            <h2 class="main-title">Main principles</h2>
+            <div class="box-content">
+                <ul>
+                    <li>
+                        A Container runtime as the <strong>only one</strong> dependency in developer laptop.
```
Comment (achedeuzot) [inline index.html]: only (remove "one")

Comment (achedeuzot) [reply index.html]: on the developer's laptop.

File: index.html [hunk 3]
```diff
@@ -1,3 +1,130 @@
...
+                        <p class="details">
+                            Docker provides a full integration with <a href="https://www.docker.com/docker-mac">Mac</a>, <a href="https://www.docker.com/docker-windows">Windows</a> & <a href="https://www.docker.com/docker-ubuntu">Linux <em>(ubuntu)</em></a>.
```
Comment (achedeuzot) [inline index.html]: update `&` to `&amp;`

File: index.html [hunk 4]
```diff
@@ -1,3 +1,130 @@
...
+                        </p>
+                    </li>
+                    <li>
+                        Use generic tasks name to launch common actions.
```
Comment (achedeuzot) [inline index.html]: task names (and not tasks name)

File: index.html [hunk 5]
```diff
@@ -1,3 +1,130 @@
...
+                    </li>
+                    <li>
+                        Let developers use the technologies & library they want.
```
Comment (achedeuzot) [inline index.html]: libraries

File: index.html [hunk 6]
```diff
@@ -1,3 +1,130 @@
...
+                    </li>
+                    <li>
+                        Simplify technical on-boarding.
+                    </li>
+                    <li>
+                        Simplify projects switching.
```
Comment (achedeuzot) [inline index.html]: project (no S)

File: index.html [hunk 7]
```diff
@@ -1,3 +1,130 @@
...
+                </ul>
+            </div>
+        </div>
+        <div class="box box-grey">
+            <h2 class="main-title"><em>Makefile</em> specification</h2>
+            <ul class="make-specification">
+                <li>
+                    <span>make checkstyle</span>
+                    <p class="details">
+                        Check lint, checkstyle rules. e.g. <a href="https://www.pylint.org/">pylint</a>, <a href="https://github.com/squizlabs/PHP_CodeSniffer">phpcs</a>, <a href="https://eslint.org/">eslint</a>, <a href="https://github.com/checkstyle/checkstyle">checkstyle <em>(java)</em></a> etc ...
```
Comment (achedeuzot) [inline index.html]: code styling rules (not checkstyle, it's not an english word)

File: index.html [hunk 8]
```diff
@@ -1,3 +1,130 @@
...
+                <li>
+                    <span>make checkstyle</span>
+                    <p class="details">
+                        Check lint, checkstyle rules. e.g. <a href="https://www.pylint.org/">pylint</a>, <a href="https://github.com/squizlabs/PHP_CodeSniffer">phpcs</a>, <a href="https://eslint.org/">eslint</a>, <a href="https://github.com/checkstyle/checkstyle">checkstyle <em>(java)</em></a> etc ... 
+                    </p>
+                </li>
+                <li>
+                    <span>make complexity</span>
+                    <p class="details">
+                        Ciclomatic complexity check <em><a href="https://en.wikipedia.org/wiki/Cyclomatic_complexity">(McCabe)</a></em>, <a href="https://pypi.python.org/pypi/radon">radon <em>(python)</em></a>, <a href="https://eslint.org/docs/rules/complexity">eslint (js)</a>, <a href="https://phpmd.org/">PHPMD</a>, <a href="http://www.scalastyle.org/rules-0.5.0.html">rules</a> <em>(scala)</em> etc ...
```
Comment (achedeuzot) [inline index.html]: Cyclomatic complexity (there's a 'y' in cycle, not cicle)

File: index.html [hunk 9]
```diff
@@ -1,3 +1,130 @@
...
+                <li>
+                    <span>make test-functional</span>
+                </li>
+                <li>
+                    <span>make test-integration</span>
+                </li>
+                <li>
+                    <span>make run</span>
+                    <p class="details">
+                        Run locally the application, e.g. node index.js, python -m myapp, go run mypapp etc ...
```
Comment (achedeuzot) [inline index.html]: Locally run the application

Comment (achedeuzot) [reply index.html]: go run myapp (not mypapp)

File: index.html [hunk 10]
```diff
@@ -1,3 +1,130 @@
...
+                <li>
+                    <span>make run</span>
+                    <p class="details">
+                        Run locally the application, e.g. node index.js, python -m myapp, go run mypapp etc ... 
+                    </p>
+                </li>
+                <li>
+                    <span>make watch</span>
+                    <p class="details">
+                        Hot reloading for development use.
```
Comment (achedeuzot) [inline index.html]: for development. (no need for "use")

File: index.html [hunk 11]
```diff
@@ -1,3 +1,130 @@
...
+                    </p>
+                </li>
+            </ul>
+        </div>
+    </section>
+    <section>
+        <h2>Story</h2>
+        <p>
+            Developers live in a technological world which is continuously changing.<br />
+            Projects switching, technologies switching, languages switching, a big bunch of switching, don't it ?<br />
```
Comment (achedeuzot) [inline index.html]: Project switching (no S to project), technology switching (same),
language switching (same), 

a lot of switching isn't it ?

File: index.html [hunk 12]
```diff
@@ -1,3 +1,130 @@
...
+            At dailymotion, to take care of the mental health of our engineers, as they are working over 500+ repositories,
```
Comment (achedeuzot) [inline index.html]: At dailymotion, we take care of the mental health of our engineers, as they work on over 500+ repositories,
developed with more than 50 technological stacks. Therefore we decided to re-think our way to deal with the complexity of our internal codebase.

**Review by achedeuzot — CHANGES_REQUESTED** [review]

File: index.html (hunk 6 above)
Comment (abusi) [reply index.html]: IMHO `Simplify switching between projects`

File: index.html (hunk 11 above)
Comment (abusi) [reply index.html]: Héhé, Bretigny style :p

File: index.html [hunk 13]
```diff
@@ -1,3 +1,130 @@
...
+                <li>
+                    <span>make complexity</span>
+                    <p class="details">
+                        Ciclomatic complexity check <em><a href="https://en.wikipedia.org/wiki/Cyclomatic_complexity">(McCabe)</a></em>, <a href="https://pypi.python.org/pypi/radon">radon <em>(python)</em></a>, <a href="https://eslint.org/docs/rules/complexity">eslint (js)</a>, <a href="https://phpmd.org/">PHPMD</a>, <a href="http://www.scalastyle.org/rules-0.5.0.html">rules</a> <em>(scala)</em> etc ...
+                    </p>
+                </li>
+                <li>
+                    <span>make fmt</span>
+                    <p class="details">
+                        Format code. e.g <a href="https://prettier.io/">Prettier <em>(js)</em></a>, <a href="https://golang.org/pkg/fmt/">fmt <em>(golang)</em></a>
```
Comment (fpietka) [inline index.html:87]: From PHPCodeSniffer you can use phpcbf to fix some (but not all) rules automatically (indent, alignments, etc.)

File: index.html (hunk 1 above)
Comment (tsunammis) [reply index.html]: Fixed

File: index.html (hunk 2 above)
Comment (tsunammis) [reply index.html]: fixed

File: index.html (hunk 3 above)
Comment (tsunammis) [reply index.html]: fixed

File: index.html (hunk 4 above)
Comment (tsunammis) [reply index.html]: fixed

File: index.html (hunk 5 above)
Comment (tsunammis) [reply index.html]: Fixed

File: index.html (hunk 6 above)
Comment (tsunammis) [reply index.html]: fixed

File: index.html (hunk 7 above)
Comment (tsunammis) [reply index.html]: Fixed

File: index.html (hunk 8 above)
Comment (tsunammis) [reply index.html]: fixed

File: index.html (hunk 9 above)
Comment (tsunammis) [reply index.html]: Fixed

File: index.html (hunk 10 above)
Comment (tsunammis) [reply index.html]: Fixed

**Review by abusi — APPROVED** [review]

**Pull request merged**